docs/examples/viz_markers.py

```diff
@@ -0,0 +1,57 @@
+"""
+======================================================================
+Fury Markers
+======================================================================
+
+This it's a example which shows how to use the marker actor.
+"""
+import numpy as np
+from fury import actor, window
+n = 10000
+
+###############################################################################
+# The are nine types 2d markers: circle, square, diamond, triangle, pentagon, 
+# hexagon, heptagon, cross and plus.
+
+marker_symbols = ['o', 's', 'd', '^', 'p', 'h', 's6', 'x', '+']
+markers = [
+    np.random.choice(marker_symbols)
+    for i in range(n)]
+
+centers = np.random.normal(size=(n, 3), scale=10)
+
+colors = np.random.uniform(size=(n, 3))
+
+############################################################################
+# You can control the edge color and edge width for each marke
+
+nodes_actor = actor.markers(
+    centers,
+    marker=markers,
+    edge_width=.1,
+    edge_color=[255, 255, 0],
+    colors=colors,
+    scales=.5,
+)
+
+############################################################################
+# In addtion, the 3d impostor sphere it's also a valid type of marker
```

Comment:
typo `it's`->`is`

maybe not mention impostor here?

Reply:
short reminder concerning this comment. Also, there is a typo in `addition`

Reply:
fixed